__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  it("responds with an array with 'slug' and 'description' properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        const topics = res.body.topics;
+        expect(Array.isArray(topics)).toEqual(true);
+        expect(topics.length).toBeGreaterThan(0);
```

Comment:
I believe you can be more specific about the length check here. We know the test data will have a length of XXX so why not be specific